### DISCUSSION TITLE
Is there any way to validate the Pocketbase autentication using an external backend?

### DISCUSSION THREAD

**Discussion by Verzach3** [pocketbase/pocketbase#4986] [Q&A] [answered]
I am going to create an application and I want to use pocketbase for the simplicity of authentication and data collection, plus I can host it by myself easily, but I also need custom functionality, so I want to create my own microservices and authenticate the users with the pocketbase JWT, in services like Supabase there are functions like getUser that receive the JWT and check if it is valid from the server side, is there any way to do this with pocketbase?

**Comment by ganigeorgiev** [comment, marked as answer]
You can find the available auth Web APIs in https://pocketbase.io/docs/api-records/#auth-record-actions.

Sending [Auth refresh](https://pocketbase.io/docs/api-records/#auth-refresh) requests will validate the `Authorization: TOKEN` header and on success will return a new token with updated `exp` claim and the latest user data. On invalid or expired token returns an error response.

You can also use the [View record API](https://pocketbase.io/docs/api-records/#view-record) (`getOne`) for the specific user but this depends on your `users` collection View API rule.

**Reply by Verzach3** [reply]
Thank you very much
